mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.kt

```diff
@@ -75,10 +84,26 @@ class LinkBankAccountActivity : BaseActivity(), LinkBankAccountView {
     private var banksList: ArrayList<Bank>? = null
     private var popularBanks: ArrayList<Bank>? = null
     private var bankSelected: String? = null
+
+    private lateinit var binding: ActivityLinkBankAccountBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_link_bank_account)
-        ButterKnife.bind(this)
+        binding = ActivityLinkBankAccountBinding.inflate(layoutInflater)
```

Comment:
remove binding from the activities/fragments while refactoring , we want to migrate all of the screens to compose